lib/virt_autotest/common.pm

```diff
@@ -243,6 +264,9 @@ if (get_var("REGRESSION", '') =~ /xen/) {
         sles12sp5 => {
             name => 'sles12sp5',
         },
+        sles12sp5ES => {
```

Comment:
If all the vm names of 12sp5 ltss-es can align with this one, it will be be more aligned in naming style ;). This is optional ofc.